backend/app/schemas/__init__.py

```
@@ -0,0 +1,17 @@
+from .user import *
```

Ruff (0.11.9)

`from .user import *` used; unable to detect undefined names

(F403)